sputnik-staking/src/user.rs

```diff
@@ -53,12 +51,12 @@ impl User {
 
     fn assert_storage(&self) {
         assert!(
-            (self.storage_used as Balance) * env::storage_byte_cost() <= self.near_amount.0,
+            env::storage_byte_cost().saturating_mul(self.storage_used as u128) <= self.near_amount,
             "ERR_NOT_ENOUGH_STORAGE"
         );
     }
 
-    pub(crate) fn delegated_amount(&self) -> Balance {
+    pub(crate) fn delegated_amount(&self) -> u128 {
```

Comment:
Can we use NearToken here and in all delegate* / undelegate / withdraw methods? (Check all `Balance` instances)